### PR TITLE
Add Partners page

### PR DESCRIPTION
**Pull request by Shelley-BaoYue**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

feature

* **What is the new behavior (if this is a feature change)?**

Add a partner page, and instructions.

### CODE REVIEW

File: src/pages/partners/partners.mdx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: KubeEdge Partners
+hide_table_of_contents: true
+---
+
+<span style={{fontSize:'18px', display:'block', marginBottom:'10px'}}>KubeEdge works with partners to create a strong, vibrant edge computing ecosystem. Here we have a list of <span style={{fontWeight:'bold'}}>KubeEdge Distributions, Hosted Platforms and Installers</span> with deep experience helping enterprises successfully adopt KubeEdge.</span>
```
Comment (fisherxu) [inline src/pages/partners/partners.mdx:6]:
```suggestion
<span style={{fontSize:'18px', display:'block', marginBottom:'10px'}}>KubeEdge works with partners to create a strong, vibrant cloud native edge computing ecosystem. Here we have a list of <span style={{fontWeight:'bold'}}>KubeEdge Distributions, Hosted Platforms and Installers</span> with deep experience helping enterprises successfully adopt KubeEdge.</span>
```

**Review by fisherxu — COMMENTED** [review]
Looks good :)

File: src/pages/partners/partners.mdx (hunk 1 above)
Comment (Shelley-BaoYue) [reply src/pages/partners/partners.mdx]: > 

done.

**Review by fisherxu — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**